packages/shared/src/css/styles.css

```diff
@@ -1,6 +1,7 @@
 :root {
   --p-transition-duration: 0s;
```

Comment:
--pds-xxx for public css vars
--_pds-xxx for internal css var (if possible with leading underscore)
- internal css vars should be removed from prod build
- add a unit test which ensures no internal css vars are present